pkg/maglev/maglev.go

```diff
@@ -99,7 +119,7 @@ func getPermutation(backends []string, m uint64, numCPU int) []uint64 {
 	}
 	wg.Wait()
 
-	return perm
+	return permutation
```

Comment:
Maybe slice `permutation` to `len(backends)*m` here in case it was allocated with a larger `len(backends)` in a previous call to `getPermutation`? Or can that never be the case?
```suggestion
	return permutation[:len(backends)*m]
```

Reply:
Thanks! This is a great suggestion as it allows me to get rid of new test code I wrote just to assert on the slice.